index.js

```diff
@@ -6,112 +6,147 @@ continueDisplayDog.style.display = "none";
 var continueDisplayLlama = document.getElementById('continueLlama');
 continueDisplayLlama.style.display = "none";
```

Comment:
Getting errors on other pages as you are still trying to set styles on elements that don't exist. You should only need the code in the if statements for changing style and not lines 3, 5, and 7.

![Screen Shot 2021-08-12 at 11 42 07 AM](https://user-images.githubusercontent.com/292375/129235424-49ec7998-784f-45b7-afd6-2df20da915ef.png)